lib/surface_site_web/live/contexts/example01.ex

```diff
@@ -4,8 +4,8 @@ defmodule SurfaceSiteWeb.Contexts.Example01 do
     import Ecto.Changeset
 
     schema "users" do
-      field :email, :string
-      field :name, :string
+      field(:email, :string)
+      field(:name, :string)
```

Comment:
Was that the formatter? If so, you could try to add `:ecto` to https://github.com/surface-ui/surface_site/blob/3dca3829635d684f530c2be8fdf98b69340a9aea/.formatter.exs#L2

Reply:
Yes will do that, didn't even cross my mind to check the formatter config 😅